### DISCUSSION TITLE
Are there any users for the RGEDI workspace ?

### DISCUSSION THREAD

**Discussion by sujen1412** [MAAP-Project/Community#786] [Platform]
Do we have any more users who are using the rgedi image?
The platform team would like to remove it from the new workspace stacks if there is no use for it.

**Comment by wildintellect** [comment]
Abigail has expressed interest in using it but we might need to reach out to the author (Carlos Silva) of the rGEDI package in R about upgrading the dependencies. cc: @lauraduncanson
https://github.com/carlos-alberto-silva/rGEDI

**Reply by wildintellect** [reply]
Relevant discussion of the dependency changes https://geocompx.org/post/2023/rgdal-retirement/
